packages/react/src/components/Select/index.tsx

```diff
@@ -14,19 +14,74 @@ export interface SelectProps
   label: string;
   options?: SelectOption[];
   children?: React.ReactElement<HTMLOptionElement | HTMLOptGroupElement>[];
+  value?: any;
```

Comment:
I'm copying [material-ui](https://material-ui.com/api/select/#props) in allowing `any` on `value` and `defaultValue`